src/web/components/Onwards/OnwardsUpper.tsx

```diff
@@ -70,6 +71,116 @@ const onwardsWrapper = css`
 	width: 100%;
 `;
 
+const headlinesContainer = (edition: Edition): string => {
```

Comment:
These container functions all seem do the same thing - would it be worth abstracting this and passing a single function the type (eg 'headlines', 'sport', 'opinion' etc) and have it make the regional selection accordingly - maybe by accessing an object with the data?

Reply:
great idea - please check now and lmk your thoughts :)